HTML/EN/settings/server/basic.html

```diff
@@ -93,4 +93,10 @@
 		<a id="progressLink" href="[% webroot %]settings/server/status.html?player=[% playerid %]">[% IF scanning %][% "SETUP_VIEW_SCANNING" | string %][% ELSE %][% "SETUP_VIEW_NOT_SCANNING" | string %][% END %]</a>
 	[% END %]
 
+	[% IF can_restart %]
+		[% WRAPPER setting title="SETUP_RESTART" desc="SETUP_RESTART_DESC" %]
+			<input name="pref_restart" type="submit" class="stdclick disableOnScan" value="[% "SETUP_RESTART_BUTTON" | string %]" [% IF scanning %]disabled[% END %]>
```

Comment:
Oh, one more thing: please don't use the `pref_` prefix for this element, as those are considered preferences. The base class would use them to automatically store updated prefs. But this isn't a pre.

Reply:
Ok, but note that the "Rescan" button also uses the "pref_[rescan]" prefix.